File: week-1/problem-set/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int number;
    do {
        number = get_int("Height: ");
    } while (number <  1);

    for (int i = number; i > 0; i--) {
        for (int j = 0; j < number; j++) {
            if (j < i-1) {
                printf(" ");
            } else {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: week-6/problem-set/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    num_height = 0
    height = ""
    while (num_height > 8) or (num_height <= 0):
        height = get_int("Height: ")
        num_height = int(height)

    for i in range(num_height):
        print((" " * (int(height) - (i + 1)) + "#" * (i + 1)) + ("  ") + "#" * (i + 1))


if __name__ == "__main__":
    main()
